lesson_12 new/contact_us.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Lesson_12</title>
+    <link rel="stylesheet" href="styles/main.css" type="text/css" />
+    <link rel="stylesheet" href="styles/contact_us.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
and again what font import here?
it's just link to Google Fonts without any font inside